### DISCUSSION TITLE
Problem in Excercise - Create a website hosted in Azure

### DISCUSSION THREAD

**Discussion by pradoshs0414** [github/india#347] [Students]
**Instead of getting these options while creating the WordPress website. As shown in the image below**

![image](https://user-images.githubusercontent.com/74011245/136602949-a2b7310a-9bcf-46ec-bca8-de04bfdc2938.png)

 **I am getting options that are not shown in the docs at MS learn. As shown in the image below**

![image](https://user-images.githubusercontent.com/74011245/136603191-c711fe3d-1acc-4dd0-8db3-c921fa88f49b.png)

Please help me out and tell me what should I do about this problem

**Comment by TheSuranaverse** [comment]
A group for all to get notified for any **Official FRT Announcement, Discuss & Clear any Doubts, Learn & Complete** together.
One-stop place for daily **Events/Workshops + Hackathons + Internships/Jobs/Free Scholarship & Courses + 24/7 Live Music** & much more...

Link to join: https://discord.gg/ceMXzhfaka

Thank you 🙏
